### PR TITLE
EOS-13713: Fix systemdwatchdog sensor for disk alerts

### PR DESCRIPTION
**Pull request by sandeepsgt**
- use common config for node_id, cluster_id, site_id
- fix bug in interface_added function

## Problem Statement
<pre>
  <code>
    https://jts.seagate.com/browse/EOS-13713
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

File: low-level/sensors/impl/centos_7/systemd_watchdog.py [hunk 1]
```diff
@@ -943,6 +945,7 @@ def _interface_added(self, object_path, interfaces_and_properties):
 
                 with self._drive_info_lock:
                     # Update drives
+                    self._drives[object_path] = {}
```
Comment (cdeshmukh) [inline low-level/sensors/impl/centos_7/systemd_watchdog.py:948]: this is only line fixing the issue ?

File: low-level/sensors/impl/centos_7/systemd_watchdog.py [hunk 2]
```diff
@@ -56,6 +56,8 @@
 from dbus.mainloop.glib import DBusGMainLoop
 import socket
 
+store = file_store
```
Comment (cdeshmukh) [inline low-level/sensors/impl/centos_7/systemd_watchdog.py:59]: reason ? node disks data status storage seems logical to reside on same server and not needed on node replacement. however any functional issue you saw without this ?

File: low-level/sensors/impl/centos_7/systemd_watchdog.py [hunk 3]
```diff
@@ -925,6 +926,7 @@ def _interface_added(self, object_path, interfaces_and_properties):
         try:
             self._log_debug("Interface Added")
```
Comment (cdeshmukh) [inline low-level/sensors/impl/centos_7/systemd_watchdog.py:927]: merge these logs & reduce logging. same for line 929, 976

File: low-level/sensors/impl/centos_7/systemd_watchdog.py (hunk 1 above)
Comment (sandeepsgt) [reply low-level/sensors/impl/centos_7/systemd_watchdog.py:948]: Yes, `self._drives[object_path]` was not initialized before storing drive info
`self._drives[object_path][self.DRIVE_DBUS_INFO] = interfaces_and_properties['org.freedesktop.UDisks2.Drive']`

Comment (cdeshmukh) [reply low-level/sensors/impl/centos_7/systemd_watchdog.py:948]: ok

File: low-level/sensors/impl/centos_7/systemd_watchdog.py (hunk 2 above)
Comment (sandeepsgt) [reply low-level/sensors/impl/centos_7/systemd_watchdog.py:59]: If node is replaced, consul will still have old drive's data. That's why it is changed to file store

**Review by saurabhkhanvilkar — APPROVED** [review]

**Review by thavanathan — APPROVED** [review]
Looks good...

File: low-level/sensors/impl/centos_7/systemd_watchdog.py (hunk 3 above)
Comment (sandeepsgt) [reply low-level/sensors/impl/centos_7/systemd_watchdog.py]: Done

File: low-level/sensors/impl/centos_7/systemd_watchdog.py (hunk 2 above)
Comment (cdeshmukh) [reply low-level/sensors/impl/centos_7/systemd_watchdog.py:59]: ok, so same reason

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**